www/js/toolbar.js

```diff
@@ -94,6 +94,34 @@ const setListeners = () => {
             type: 'response-control',
             success: isPinCorrect,
           });
+
+          // if Pin is correct and there is a shabad already in desktop, emit that shabad details.
+          if (isPinCorrect) {
+            const currentShabad = global.core.search.getCurrentShabadId();
+            const currentVerse = document.querySelector(`#shabad .panktee.current`);
+            const homeVerse = document.querySelector(`#shabad .panktee.main`);
```

Comment:
I'm guessing updating the home line from desktop is out of scope right now?

Reply:
Not too worried about that if we can't add it right now